ruby-connector-v2-sample/app/jobs/callbacks/session_successer.rb

```diff
@@ -0,0 +1,17 @@
+class Callbacks::SessionSuccesser < ApplicationJob
```

Comment:
I suggest a better naming, something like ` Callbacks::SessionSuccess`. Same for Updater.

Reply:
I prefer using nouns for class namings and verbs for methods